src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.registrationexception.RegistrationException;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_UserIsValid_Ok() {
+        User user = new User("Arsenal", "Ar789al", 18);
+        assertEquals(user, registrationService.register(user));
+    }
+
+    @Test
+    void register_ExistingUser_NotOk() {
+        User user = new User("Arsenal", "Ar789al", 18);
+        Storage.PEOPLE.add(user);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_NullLogin_NotOk() {
+        User user = new User(null, "validity", 18);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_NullPassword_NotOk() {
+        User user = new User("Arsenal", null, 18);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void register_NullAge_NotOK() {
+        User user = new User("Arsenal", "Ar789al", 0);
+        assertThrows(RegistrationException.class, () -> registrationService.register(user));
+    }
+
+    @Test
+    void user_NotEnoughLoginLength_NotOk() {
```

Comment:
be consistent for test method naming. 
register_LoginLengthLessThenMinLimit_NotOk 

just an example

Check all places